### ISSUE TITLE
Disconnected roads at zooms < 6

### ISSUE THREAD

**Issue by dnomadb** [protomaps/basemaps#213]
**Location**
https://maps.protomaps.com/#map=5.5/42.624/-90.609&theme=light&renderer=maplibregl&tiles=https://build.protomaps.com/20240131.pmtiles

**Screenshots**
<img width="946" alt="image" src="https://github.com/protomaps/basemaps/assets/5084513/300af962-5483-461b-8cea-6320028538ec">
<img width="870" alt="image" src="https://github.com/protomaps/basemaps/assets/5084513/b78e7fc6-0601-440a-8755-3f2905a6b16d">

**Required information**
 - Tileset version and source: `https://build.protomaps.com/20240131.pmtiles` (but also present in earlier builds)

**Comment by nvkelso** [comment]
If we could detect these are part of bigger road relations, we could bring the missing pieces (that aren't motorways) earlier?

**Comment by bdon** [comment]
Yeah, I think highway relations are the only way to solve this, because right now it's using a naive interpretation of `highway=motorway` to generalize. A approach similar to what we're doing with minAdminLevel in the boundaries layer could work.

**Comment by wipfli** [comment]
The `ref` tag in OSM can also help maybe. At least would for A2 which turns from  `highway=motorway` to `highway=trunk` in the Gotthard tunnel in Switzerland... https://maps.protomaps.com/#map=9.47/46.6034/8.6464&theme=light&renderer=maplibregl&tiles=https://build.protomaps.com/20240131.pmtiles

**Comment by wipfli** [comment]
In the US this was fixed by #386. There are other places where roads are not continuous like Bretagne in France but let's close this issue for now. Feel free to share more locations @dnomadb
